src/stories/Dialog.stories.tsx

```diff
@@ -25,42 +25,18 @@ const TemplateMessage: StoryFn<DialogProps> = (args) => {
   const [dialogVisibility, setDialogVisibility] = useState(
     args.visibility ?? false
   );
+  const [templateMessage, setTemplateMessage] = useState<boolean>(false);
 
   useEffect(() => {
     setDialogVisibility(args.visibility ?? false);
   }, [args.visibility]);
-
-  return (
-    <div
-      style={{
-        position: 'absolute',
-        left: '45%',
-        top: '50%'
-      }}
-    >
-      <Button
-        sx={{ bottom: '18px' }}
-        onClick={() => setDialogVisibility(!dialogVisibility)}
-      >
-        Show Dialog
-      </Button>
-      <Dialog
-        {...args}
-        visibility={dialogVisibility}
-        setVisibility={setDialogVisibility}
-      />
-    </div>
-  );
-};
-
-const TemplateOption: StoryFn<DialogProps> = (args) => {
-  const [dialogVisibility, setDialogVisibility] = useState(
-    args.visibility ?? false
-  );
-
   useEffect(() => {
-    setDialogVisibility(args.visibility ?? false);
-  }, [args.visibility]);
+    if (args.type === 'information' || args.type === 'alert') {
```

Comment:
This condition cannot replace the use of templateMessate state? Avoiding creating a new state and useEffect.

Reply:
> now that condition render a dialog with setVisibility or accept/reject function